uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/passengerdetails/PassengerDetailsPresenter.kt

```diff
@@ -54,4 +65,75 @@ class PassengerDetailsPresenter(view: PassengerDetailsContract.View) : BasePrese
         passengerDetails = passengerDetails?.copy(firstName = firstName, lastName = lastName,
                                                   email = email, phoneNumber = mobilePhoneNumber)
     }
+
+    override fun getCountryCode(context: Context): String {
+        var countryDialingCode = getCountryDialingCodeFromNumber(passengerDetails?.phoneNumber,
+                                                                 context.resources)
+
+        if (countryDialingCode.isNotEmpty() && countryDialingCode.subSequence(0, 1).equals("+")) {
```

Comment:
doesn't the plus sign cause a detekt lint issue? Maybe we can add it to strings as an untranslatable string, or maybe move it to the companion in a const val ?